view/adminhtml/web/config.js

```diff
@@ -21,10 +21,10 @@
     "homePageCards": ["ApplyDefaultSettingsCard", "PluginSpecificCacheTagCard", "PurgeCacheCard"],
     "moreSettingsCards": {
         "container.moresettings.security": ["SecurityLevelCard", "WAFCard"],
-        "container.moresettings.performance": ["AlwaysOnlineCard", "ImageOptimizationCard", "DevelopmentModeCard"]
+        "container.moresettings.performance": ["BypassCacheByCookieCard", "AlwaysOnlineCard", "ImageOptimizationCard", "DevelopmentModeCard"]
```

Comment:
Do we want `BypassCacheByCookieCard` at the top? The order here is also the order of the cards to be rendered